package.json

```
@@ -8,7 +8,8 @@
   "scripts": {
     "lint": "eslint src/**/*.js --fix",
     "format": "prettier --write 'src/**/*.js'",
-    "start": "node src/index.js"
+    "start": "node src/index.js",
+    "generate": "plop --plopfile ./generators/plopGenerateModule.js"
```

```suggestion
    "generate:module": "plop --plopfile ./generators/plopGenerateModule.js"
```
Vale ter o sufixo do `module`? Caso no futuro a gente venha a ter mais opções, teríamos que rever a parte de seleção.